Source/RpmAvatarCreator/Private/UI/RpmAvatarEditorUI.cpp

```diff
@@ -129,6 +174,17 @@ void URpmAvatarEditorUI::OnAssetButtonClicked(const FRpmPartnerAsset& Asset)
 			RemoveAssetSelectedPin(ERpmPartnerAssetType::Outfit);
 		}
 	}
+	if (BadgeIcon)
+	{
+		if (Asset.BadgeUrl.IsEmpty())
+		{
+			BadgeIcon->SetIconTexture(nullptr);
+		}
+		else
+		{
+			AvatarCreatorApi->DownloadImage(Asset.BadgeUrl, 64, BadgeIconDownloadCompleted);
```

Comment:
not a huge deal but the size here (64) could be an adjustable property that is set to 64 by default. This size might work for most cases but it is likely some might want to adjust it for their use case.